fvtest/compilertest/ilgen/TestIlGeneratorMethodDetails.hpp

```diff
@@ -71,9 +78,14 @@ class OMR_EXTENSIBLE IlGeneratorMethodDetails : public OMR::IlGeneratorMethodDet
                                           bool forceClassLookahead,
                                           TR_InlineBlocks *blocksToInline);
 
+#define IL_VERIFY
```

Comment:
Not a huge fan of the define in here: IMO, it should be added to the fvtest makefiles